WalletWasabi.Fluent/Views/CoinControl/SelectCoinsDialogView.axaml

```diff
@@ -0,0 +1,90 @@
+    ScrollViewer.VerticalScrollBarVisibility="Disabled">
+
+    <controls:ContentArea.Styles>
+
+      <Style Selector="TreeDataGridRow">
+        <Setter Property="Focusable" Value="False" />
+        <Setter Property="Height" Value="37.5" />
+      </Style>
+
+      <Style Selector="TreeDataGridRow /template/ Rectangle">
+        <Style.Resources>
+          <converters:IndentToBrushConverter x:Key="IndentToBrushConverter">
+            <converters:IndentToBrushConverter.Brushes>
+              <generic:List x:TypeArguments="IBrush">
+                <SolidColorBrush>Transparent</SolidColorBrush>
+                <SolidColorBrush Color="{DynamicResource SystemListLowColor}" />
+              </generic:List>
+            </converters:IndentToBrushConverter.Brushes>
+          </converters:IndentToBrushConverter>
+        </Style.Resources>
+        <Setter Property="Fill">
+          <Setter.Value>
+            <MultiBinding Converter="{StaticResource IndentToBrushConverter}">
+              <Binding Path="." />
+              <Binding Path="$parent[TreeDataGridRow].Rows" />
+            </MultiBinding>
+          </Setter.Value>
+        </Setter>
+      </Style>
+
+      <Style Selector="TreeDataGridRow ContentPresenter">
+        <Setter Property="VerticalAlignment" Value="Center" />
+      </Style>
+
+      <Style Selector="TreeDataGridTemplateCell > ContentPresenter">
+        <Setter Property="Margin" Value="5 0 10 0" />
+      </Style>
+
+      <Style Selector="TreeDataGridTextCell TextBlock">
+        <Setter Property="TextAlignment" Value="Center" />
+      </Style>
+
+      <Style Selector="treeDataGrid|TreeDataGridPrivacyTextCell">
+        <Setter Property="FontFamily" Value="{StaticResource MonospacedFont}" />
+        <Setter Property="FontSize" Value="15" />
+        <Setter Property="Margin" Value="10 0" />
+      </Style>
+
+      <Style Selector="ContentPresenter > TextBlock">
+        <Setter Property="FontFamily" Value="{StaticResource MonospacedFont}" />
+      </Style>
+
+    </controls:ContentArea.Styles>
+
+    <DockPanel>
+
+      <Panel x:CompileBindings="False" DockPanel.Dock="Bottom" Margin="0 12 0 0" TextBlock.Foreground="{DynamicResource AcrylicTrimForeground}">
+        <ProgressBar Height="25" Maximum="{Binding RequiredAmount^.Satoshi}" Value="{Binding SelectedAmount^.Satoshi, Mode=OneWay}" />
+        <TextBlock Classes="h8" Margin="10 0" VerticalAlignment="Center" Text="{Binding SelectedAmount^, Converter={x:Static converters:MoneyConverters.ToFormattedString}, StringFormat='{}{0} BTC selected'}" />
+        <TextBlock Classes="h8" Margin="10 0" HorizontalAlignment="Right" VerticalAlignment="Center" IsVisible="{Binding !EnoughSelected^}"
+                   Text="{Binding RemainingAmount^, Converter={x:Static converters:MoneyConverters.ToFormattedString}, StringFormat='{}{0} BTC remaining'}" />
+      </Panel>
+
+      <TreeDataGrid Source="{Binding Source}" CanUserSortColumns="True" />
```

Comment:
I don't see ElementFactory set anywhere for TreeDataGrid
https://github.com/zkSNACKs/WalletWasabi/blob/1f14218bde76b090ebb82d8fc0cb05d5cd40e8f4/WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml#L103-L105

Reply:
I've discussed with you (@wieslawsoltes) and this might be not necessary. Needs testing on your side.

Reply:
@wieslawsoltes Have you confirmed is this is still needed?

Reply:
It's needed! I've already added it.